controller_1.py

```diff
@@ -0,0 +1,14 @@
+from model import Model
```

Comment:
This file looks like a duplicate of `controller.py` (?) If that's the case, please delete one of them.